ironfish/src/rpc/routes/eth/ethRouter.test.ts

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { Address } from '@ethereumjs/util'
+import { useAccountFixture } from '../../../testUtilities'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+
+describe('Route eth/ethRouter', () => {
+  const routeTest = createRouteTest(false)
+
+  it('should handle eth_sendTransaction successfully', async () => {
```

Comment:
maybe also worth doing the eth_getAccount test and asserting the format of the response?